azure/defaults.go

```diff
@@ -71,7 +71,7 @@ const (
 const (
 	// DefaultWindowsOsAndVersion is the default Windows Server version to use when
 	// generating default images for Windows nodes.
-	DefaultWindowsOsAndVersion = "windows-2019"
+	DefaultWindowsOsAndVersion = "windows-2022"
```

Comment:
@marosset @jsturtevant any reason we shouldn't make this change?

Reply:
we should make this change